lib/routes/account.js

```diff
@@ -406,177 +398,48 @@ module.exports = (log, db, mailer, Password, config, customs, checkPassword, pus
         const form = request.payload
         const email = form.email
         const authPW = form.authPW
-        const service = request.payload.service || request.query.service
-        const redirectTo = request.payload.redirectTo
-        const resume = request.payload.resume
-        const ip = request.app.clientAddress
-        const requestNow = Date.now()
         const originalLoginEmail = request.payload.originalLoginEmail
         const verificationMethod = request.payload.verificationMethod || request.query.verificationMethod
+        const requestNow = Date.now()
 
-        let needsVerificationId = true
-        let accountRecord, sessions, sessionToken, keyFetchToken, mustVerifySession, doSigninConfirmation,
-          unblockCode, customsErr, didSigninUnblock, tokenVerificationId, tokenVerificationCode
-
+        let accountRecord, password, sessionToken, keyFetchToken, didSigninUnblock
         let securityEventRecency = Infinity, securityEventVerified = false
 
         request.validateMetricsContext()
 
-        // Store flowId and flowBeginTime to send in email
-        let flowId, flowBeginTime
-        if (request.payload.metricsContext) {
-          flowId = request.payload.metricsContext.flowId
-          flowBeginTime = request.payload.metricsContext.flowBeginTime
-        }
-
-        checkIsBlockForced()
-          .then(() => customs.check(request, email, 'accountLogin'))
-          .catch(extractUnblockCode)
-          .then(readEmailRecord)
-          .then(checkUnblockCode)
-          .then(checkSecurityHistory)
+        return checkCustomsAndLoadAccount()
           .then(checkEmailAndPassword)
-          .then(checkNumberOfActiveSessions)
+          .then(checkSecurityHistory)
           .then(createSessionToken)
+          .then(sendSigninNotifications)
           .then(createKeyFetchToken)
```

Comment:
These functions are now thin wrappers around their equivalents in the signinUtils module, which explains all the red in the diff of this file.